es/index.js

```diff
@@ -190,7 +194,7 @@ export default function axiosRetry(axios, defaultOptions) {
       retryCondition = isNetworkOrIdempotentRequestError,
       retryDelay = noDelay,
       shouldResetTimeout = false
-    } = getRequestOptions(config, defaultOptions);
```

Comment:
since defaultOptions never gets stored on config, this actually doesn't really seem necessary.